jme3-ios/src/main/java/com/jme3/renderer/ios/IosGL.java

```diff
@@ -717,6 +718,59 @@ public Object glFenceSync(int condition, int flags) {
     
     @Override
     public void glFramebufferTextureLayerEXT(int target, int attachment, int texture, int level, int layer) {
-        throw new UnsupportedOperationException("OpenGL ES 2 does not support texture arrays");
+        JmeIosGLES.glFramebufferTextureLayer(target, attachment, texture, level, layer);
+    }
+
+    // New methods from GL2 interface which are supported in GLES30
```

Comment:
Supported or unsupported? given that they are stubs atm

Reply:
The comment is not too clear... Methods below are those defined in GL2 which we need to implement because gles30 supports GL{Read|Write}Buffer and also 3D textures. The reason to create the stubs is that glAlphaFunc, glPointSize and glPolygonMode have no gles equivalent but are defined in GL2 interface.